.github/workflows/functional-test-noncloud.yaml

```diff
@@ -364,7 +362,6 @@ jobs:
           RADIUS_SAMPLES_REPO_ROOT: ${{ github.workspace }}/samples
           BICEP_RECIPE_REGISTRY: "${{ env.LOCAL_REGISTRY_NAME }}:${{ env.LOCAL_REGISTRY_PORT }}"
           BICEP_RECIPE_TAG_VERSION: ${{ env.BICEP_RECIPE_TAG_VERSION }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Where does [this](https://github.com/radius-project/radius/blob/main/test/testutil/testutil.go#L139) gets set?

Reply:
This **yaml** is for non-cloud tests. If you see [the references of **GH_TOKEN**](https://github.com/search?q=repo%3Aradius-project%2Fradius%20%22GetGitPAT%22&type=code), you can see that it is only being used in a cloud test.